src/main/java/com/segment/analytics/android/integrations/nielsendcr/NielsenDCRIntegration.java

```diff
@@ -332,7 +342,7 @@ private JSONObject mapSpecialKeys(
       }
 
       if (contentProperties.containsKey("airdate")) {
-        String airdate = contentProperties.getString("airdate");
+        String airdate = formatAirdate(contentProperties.getString("airdate"));
```

Comment:
Might want to do another check on `airdate` to see that it is not `null`. You are doing `containsKey` earlier but I'm not sure if `airdate` value can still be `null` despite having the key?